docs/contributing/style-guide.md

```diff
@@ -1455,6 +1504,42 @@ For clarity and search engine discoverability:
 * Use Google Analytics to make data-driven decisions.
 :::
 
+
+
+## UI Elements
+
+Not all UI elements have an intuitive name. It's the very nature of working with a constantly evolving product. Here are some examples:
+
+import UiElements from '../reuse/ui-elements.md';
```

Comment:
You might want to future-proof the ui-elements.md file by using UI examples from Kanso.

Reply:
Thank you! I'll work on that now and resubmit.

Reply:
Thank you! I've created a [separate ticket](https://sumologic.atlassian.net/browse/DOCS-355) for this task. I want to run it past you guys and the UX team.